akka-docs/src/main/paradox/cluster-usage.md

```diff
@@ -12,6 +12,49 @@ To use Akka Cluster, you must add the following dependency in your project:
   version="$akka.version$"
 }
 
+## When and where to use Akka Cluster
+
+An architectural choice you have to make is if you are going to use a microservices architecture or
```

Comment:
Seems a bit strange to start out with this discussion on a page titled when and where to use Akka Cluster.  I think the main point is actually what currently resides at the bottom of the page. Suggest starting the page instead with something like this:

When designing an application or service that will be deployed in a distributed environment, you need to address the challenges of coordination and communication. This is true whether you are working with a traditional application architecture or microservices.  There are valid reasons and use cases for building a traditional application, such as time to market and less initial complexity. However, microservices are proving to offer benefits that make it worth the initial investment of learning a new architectural pattern. (then your existing description of benefits, and the para can be removed from the bottom).

Note to team: The rest of the page seems to be devoted to microservices only. But, the other page has obviously been edited to include applications as well as services. So, it's still unclear to me on this page whether and how you would use Akka Cluster with a more traditional architecture?

Reply:
The context of this new section is at the top of the Akka Cluster documentation: https://doc.akka.io/docs/akka/current/cluster-usage.html

(yes, we know that page is too big, but restructuring of the documentation is another task)

My thinking was to present that there are two choices; microservices or traditional distributed application. That decision will influence how you should use Akka Cluster. Added that clarification and made them H3.

I thought it was better to start with microservices because that is what we recommend for most cases, but the other choice is also good if you have the right use cases and it's not something we want to glance over in half a sentence like in your suggestion.

> When designing an application or service that will be deployed in a distributed environment, you need to address the challenges of coordination and communication. This is true whether you are working with a traditional application architecture or microservices

I think this opens up for rather many other questions and concepts that would have to be explained, and I don't think the Akka Cluster documentation is the right place for that.